spec/models/user_spec.rb

```diff
@@ -1,12 +1,38 @@
 require "rails_helper"
 
 RSpec.describe User do
-  it { is_expected.to validate_presence_of(:email) }
-  it { is_expected.to validate_presence_of(:password_digest) }
+  context "validations" do
+    it { is_expected.to validate_presence_of(:email) }
+    it { is_expected.to validate_presence_of(:password_digest) }
 
-  context "uniqueness" do
-    before { create(:user) }
+    context "uniqueness" do
+      before { create(:user) }
 
-    it { is_expected.to validate_uniqueness_of(:email) }
+      it { is_expected.to validate_uniqueness_of(:email) }
+    end
+  end
+
+  context "associations" do
+    it { is_expected.to have_many(:recommendations) }
+  end
+
+  context "#recommendables" do
+    it "returns all recommendables" do
```

Comment:
Maybe the description could leave out the implementation classes and say something like "returns all recommended items"?